packages/user-interface/src/pages/task/task-page.module.scss

```diff
@@ -0,0 +1,5 @@
+:global {
```

Comment:
As discussed, not a very nice solution, but the only workable solution for now. In the future we should move away from bootstrap and start using our own form components.